doc/modules/ROOT/pages/index.adoc

```diff
@@ -1,46 +1,154 @@
-////
-This is the base template for Hazelcast integration module guides.
+:github-address: https://github.com/hazelcast-guides/kubernetes-wan
+:templates-url: templates:ROOT:page$/
+:hazelcast: Hazelcast IMDG
+:framework: Kubernetes
 
-You can clone this repository, work on it and create your guide, and then push to a new repository.
-////
 
-:github-address: [github-repo-address-of-your-guide]
+= Hazelcast WAN Replication (Enterprise Only)
 
-:templates-url: https://raw.githubusercontent.com/hazelcast-guides/adoc-templates/master
+This a complete example presenting how to set two Hazelcast clusters (deployed in two different Kubernetes environments) with the WAN Replication in between them.
 
-// Use this relative url if you are going to publish the guide on the guides site.
-// Note that this url will not work locally and raise asciidoctor errors.
-// So, complete the guide with the above url and set the below one just before 
-// publishing on the guides site.
-//
-// :templates-url: templates:ROOT:page$/
+include::{templates-url}/link-to-repo.adoc[]
 
-= Title
+== What You’ll Learn
 
-// Content entered directly below the header but before the first section heading is called the preamble.
+In this guide you will create two Hazelcast clusters and will set up WAN Replication between them.
 
-include::{templates-url}/link-to-repo.adoc[]
+== Introduction
 
-== What You’ll Learn
+This example focuses on the WAN Replication feature and assumes that you have some general knowledge about Hazelcast on Kubernetes. Here are some resources:
```

Comment:
May be a link to WAN doc or a brief explanation of it? Assume someone never heard WAN ends up with this guide. I think it would be helpful for such a situation.

Reply:
Added the link to the list of resources